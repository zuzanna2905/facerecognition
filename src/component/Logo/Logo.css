.Tilt {
    background: linear-gradient(to bottom, #DA627D 0%, #A53860 54%, #450920 100%);
}