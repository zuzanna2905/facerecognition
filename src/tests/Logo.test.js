import {shallow} from 'enzyme';
import React from 'react';
import Logo from '../component/Logo/Logo';

it('renders Logo without crashing', () => {
    let  wrapper = shallow(<Logo/>);
    expect(wrapper).toMatchSnapshot();
});