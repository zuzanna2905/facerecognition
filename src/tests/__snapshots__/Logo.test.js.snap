// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Logo without crashing 1`] = `
<div
  className="ma4 mt0"
>
  <Tilt
    className="Tilt br2 shadow-2"
    options={
      Object {
        "max": 55,
      }
    }
    style={
      Object {
        "height": 100,
        "width": 100,
      }
    }
  >
    <div
      className="Tilt-inner"
    >
      <img
        alt="logo"
        src="brain.png"
        style={
          Object {
            "paddingTop": "25px",
          }
        }
      />
    </div>
  </Tilt>
</div>
`;
